test/e2e/spec/oidc_provider/client_spec.js

```diff
@@ -0,0 +1,67 @@
+describe("OpenId Provider", function() {
+    
+    var chakram = require('/usr/local/lib/node_modules/chakram/lib/chakram.js');
+    var expect = chakram.expect;
+    var DOMAIN = 'https://' + process.env.BOILERPLATE_DOMAIN;
```

Comment:
Domains don't start with `https://` and `DOMAIN` isn't used other than in the next line, so please update the next line to 

    var URL = 'https://' + process.env.BOILERPLATE_DOMAIN + '/api/openid';